runtime/templates/bare64_main.c

```diff
@@ -0,0 +1,10 @@
+#include "isp_utils.h"
```

Comment:
These seem like duplicates of `bare_main.c` and `frtos_main.c`. If so, we don't need them.

Reply:
Will be removed with other duplicate runtime simplifications.